.github/workflows/log-viewer-docs.yaml

```
@@ -0,0 +1,57 @@
+name: "log-viewer-docs"
+
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily at 00:15 UTC (the 15 is to avoid periods of high load)
+    - cron: "15 0 * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: "${{github.workflow}}-${{github.ref}}"
+  # Cancel in-progress jobs for efficiency
```

```suggestion

  # Cancel in-progress jobs for efficiency
```

In new code, we try to include a newline before every comment block.